assets/src/scss/new-identity/style.scss

```diff
@@ -1,12 +1,18 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --box-shadow--offset-y: calc(var(--link--height) * -1);
+  --transition--duration: 0.3s;
+
   border-bottom: 2px solid var(--gp-green-500);
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+  transition: var(--transition--duration) cubic-bezier(0.5, 0, 0, 1);
 
   &:hover,
-  &:focus {
-    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+  &:active {
+    box-shadow: inset 0 var(--box-shadow--offset-y) rgba($gp-green-500, .15);
```

Comment:
FYI, After a call with Houssam, he propose to use the same `box-shadow` to all these cases.